wisl/lib/semantics/wislSHeap.ml

```diff
@@ -383,3 +408,46 @@ let add_debugger_variables
       scopes vars
   in
   scopes
+
+(***** Clean-up *****)
+
+let clean_up ?(keep = Expr.Set.empty) (heap : t) : Expr.Set.t * Expr.Set.t =
```

Comment:
This is always being called from `WislSMemory.clean_up`, which itself has a `?keep` argument
There's no point in having both of these optional
Make this one non-optional?

Reply:
Done